tests/e2e/specs/in-game/layout.spec.js

```diff
@@ -37,7 +37,7 @@ describe('Game View Layout', () => {
 
     cy.resolveOpponent();
 
-    cy.get('#waiting-for-opponent-counter-scrim').should('not.be.visible');
```

Comment:
Really not sure why this changed in this PR. Most of the time we use existence checks rather than visibility checks because most dynamic rendering we use actually removes elements from the dom rather than futzing with their visibility. So I'm not sure why this one used to work with not.be.visible, but the element is no longer in the DOM so we need to use not.exist